addons/oci-finops/finops-setup/finops_iam.auto.tfvars.json

```diff
@@ -0,0 +1,82 @@
+{
+  "compartments_configuration": {
+    "enable_delete": "true",
+    "compartments": {
```

Comment:
You'll get conflicts from this, OE already had these compartments, you want to create only the compartments the addon actually needs and then referenced them using *parent_id* or *default_parent_id*. If you use KEYs than dependency functionality can be used across the stacks to automatically replaced them with correct ocids